source/faq.rst

```diff
@@ -1,264 +1,219 @@
+By default, both Prometheus and QAN store time-series data for 30 days.
 
-Depending on available disk space and your requirements,
-you may need to adjust data retention time.
+Depending on your available disk space and requirements, you may need to adjust the data retention time.
 
 You can control data retention by the following way.
 
-#. Select the |pmm-settings| dashboard in the main menu.
+#. Select the PMM Settings dashboard in the main menu.
+
+   .. image:: /.res/graphics/png/pmm-add-instance.png
+
+#. In the *Settings* section, enter the new data retention value in days.
 
-   .. figure:: .res/graphics/png/pmm-add-instance.png
+   .. image:: /.res/graphics/png/pmm.settings_settings.png
 
-      Choosing the |pmm| *Settings* menu entry
+#. Click *Apply changes*.
 
-#. In the *Settings* section, enter new data retention value in days.
 
-   .. figure:: .res/graphics/png/pmm.settings_settings.png
 
-      Entering data retention on the *Settings dashboard*
+***********************************************
+How often are NGINX logs in PMM Server rotated?
+***********************************************
 
-#. Click the *Apply changes* button.
+PMM Server runs ``logrotate`` on a daily basis to rotate NGINX logs and keeps up to ten of the most recent log files.
 
-How often are nginx logs in PMM Server rotated?
-================================================================================
 
-|pmm-server| runs ``logrotate`` to rotate nginx logs on a daily basis
-and keep up to 10 latest log files.
 
 .. _privileges:
 
-What privileges are required to monitor a |mysql| instance?
-================================================================================
+*********************************************************
+What privileges are required to monitor a MySQL instance?
+*********************************************************
+
+.. code-block::
+
+   GRANT SELECT, PROCESS, SUPER, REPLICATION CLIENT, RELOAD ON *.* TO 'pmm'@'localhost';
+   GRANT SELECT ON performance_schema.* TO 'pmm'@'localhost';
+
+(See :ref:`pmm.conf-mysql.user-account.creating`.)
 
-See :ref:`pmm.conf-mysql.user-account.creating`.
 
+
+*****************************************
 Can I monitor multiple service instances?
-================================================================================
+*****************************************
 
-Yes, you can add multiple instances of |mysql| or some other service to be
-monitored from one |pmm-client|. In this case, you will need to provide a
-distinct port and socket for each instance, and specify a unique name for each
-instance (by default, it uses the name of the |pmm-client| host).
+You can add multiple instances of MySQL or some other service to be monitored from one PMM Client. In this case, you must provide a unique port or socket for each instance, and specify a unique name for each.  (If a name is not provided, PMM uses the name of the PMM Client host.)
```

Comment:
or IP address for a multi homed instance.  There's three optiosn for running pmm2-client against multiple local instances.